### PR TITLE
workaround ADT bug causing only 1 object being checked by ATC

### PR DESCRIPTION
**Pull request by gtreskas**
Multiple atc packages checks is currently not possible with one call in ADT, because the ADT team is not fixing this bug we took the chance to make not rely on the availability of such feature in ADT, and make sure we execute them one by one and accumulate the results back to a single report.

### CODE REVIEW

File: sap/cli/atc.py [hunk 1]
```diff
@@ -201,8 +201,12 @@ def run(connection, args):
         args.variant = settings.system_check_variant
         
     results = []
-    if args.name is not None and isinstance(args.name, list) and len(args.name) > 0:
-        arr_objects = args.name[0].split()
+    if args.name and isinstance(args.name, list):
+        arr_objects = []
+        # Make sure it can be used as a sequence of strings or as an array of strings
+        for objectArr in args.name:
+            arr_objects += objectArr.split()
```
Comment (jfilak) [inline sap/cli/atc.py:208]: I have absolutely no idea why call `objecArr.split()`.

**Review by jfilak — CHANGES_REQUESTED** [review]

File: sap/cli/atc.py (hunk 1 above)
Comment (jfilak) [reply sap/cli/atc.py:208]: You must add a test case which will test this construction.

Comment (gtreskas) [reply sap/cli/atc.py:208]: They way we are calling sapcli from our DDCI pipeline is: "APPL_MM_PUR_MPS VDM_MM_PUR_MPS", this means actually we have to parse 2 packages and split them into 2 array records so we can execute them seperately.

Will add a test case.

Comment (jfilak) [reply sap/cli/atc.py:208]: args.name is already a list -> https://github.com/jfilak/sapcli/blob/master/sap/cli/atc.py#L156 - (see nargs='+')

Comment (gtreskas) [reply sap/cli/atc.py:208]: done. Can you recheck?

File: test/unit/test_sap_cli_atc.py [hunk 2]
```diff
@@ -163,6 +163,21 @@ def test_package_multiple(self, fake_object, fake_fetch_customizing, fake_runner
...
+    @patch('sap.cli.atc.print_worklists_to_stream')
+    @patch('sap.adt.objects.ADTObjectSets')
+    @patch('sap.adt.atc.ChecksRunner')
+    @patch('sap.adt.atc.fetch_customizing')
+    @patch('sap.adt.Package')
+    def test_package_multiple_trim(self, fake_object, fake_fetch_customizing, fake_runner, fake_sets, fake_print):
+
+        self.setUpRunMocks(fake_object, '$PACKAGE', fake_fetch_customizing, fake_runner, fake_sets)
+
+        args = parse_args('run', 'package', '$PACKAGE $TMP')
```
Comment (jfilak) [inline test/unit/test_sap_cli_atc.py:175]:
```suggestion
        args = parse_args('run', 'package', '$PACKAGE', '$TMP')
```

That's the correct line.

**Review by jfilak — CHANGES_REQUESTED** [review]

**Comment by jfilak** [comment]
Thank you! I pushed 98a39662ec66e0250f7de8a55575b084440fb94b

**Pull request closed without merging**